bundles/org.openhab.binding.kodi/src/main/resources/ESH-INF/thing/thing-types.xml

```diff
@@ -24,6 +24,7 @@
 			<channel id="inputaction" typeId="inputaction" />
 			<channel id="systemcommand" typeId="systemcommand" />
 			<channel id="title" typeId="system.media-title" />
+            <channel id="originaltitle" typeId="system.media-title" />
```

Comment:
Can you reformat the thing-type.xml (Ctrl-Shift-F in eclipse) The indentation isn't correct (mixed spaces and tabs)

Reply:
Should I use tab or use 4 spaces?

Reply:
I have fix spaces and tabs.